src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/ExportFileController.cs

```diff
@@ -57,19 +57,19 @@ public async Task<IActionResult> Execute(int id)
                 archiveFileName = PathExtensions.Combine(Path.GetTempPath(), filename);
 
                 var recipeDescriptor = new RecipeDescriptor();
-                CustomFileDeploymentStep customFileDeploymentStep = deploymentPlan.DeploymentSteps.FirstOrDefault(ds => ds.Name == "CustomFileDeploymentStep") as CustomFileDeploymentStep;
+                RecipeFileDeploymentStep recipeFileDeploymentStep = deploymentPlan.DeploymentSteps.FirstOrDefault(ds => ds.Name == "RecipeFileDeploymentStep") as RecipeFileDeploymentStep;
 
-                if (customFileDeploymentStep != null)
+                if (recipeFileDeploymentStep != null)
                 {
-                    recipeDescriptor.Name = customFileDeploymentStep.RecipeName;
-                    recipeDescriptor.DisplayName = customFileDeploymentStep.DisplayName;
-                    recipeDescriptor.Description = customFileDeploymentStep.Description;
-                    recipeDescriptor.Author = customFileDeploymentStep.Author;
-                    recipeDescriptor.WebSite = customFileDeploymentStep.WebSite;
-                    recipeDescriptor.Version = customFileDeploymentStep.Version;
-                    recipeDescriptor.IsSetupRecipe = customFileDeploymentStep.IsSetupRecipe;
-                    recipeDescriptor.Categories = (customFileDeploymentStep.Categories ?? "").Split(',', StringSplitOptions.RemoveEmptyEntries);
-                    recipeDescriptor.Tags = (customFileDeploymentStep.Tags ?? "").Split(',', StringSplitOptions.RemoveEmptyEntries);
+                    recipeDescriptor.Name = recipeFileDeploymentStep.RecipeName;
+                    recipeDescriptor.DisplayName = recipeFileDeploymentStep.DisplayName;
+                    recipeDescriptor.Description = recipeFileDeploymentStep.Description;
+                    recipeDescriptor.Author = recipeFileDeploymentStep.Author;
+                    recipeDescriptor.WebSite = recipeFileDeploymentStep.WebSite;
+                    recipeDescriptor.Version = recipeFileDeploymentStep.Version;
+                    recipeDescriptor.IsSetupRecipe = recipeFileDeploymentStep.IsSetupRecipe;
+                    recipeDescriptor.Categories = (recipeFileDeploymentStep.Categories ?? "").Split(',', StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Could we return `Array.Empty` instead of doing split twice?

Reply:
Please give me the exact change you would like.
At least, this line ensures there is no null value and no extra space or comma.

Reply:
Another question how the `Categories` declared as `String` while the `Split` returns array of string?!!